src/renderer/components/wallet/keystore/ImportKeystore.tsx

```diff
@@ -81,7 +81,10 @@ export const ImportKeystore: React.FC<Props> = (props): JSX.Element => {
   const uploadKeystore = () => {
     unsubScribeKeystoreSub()
     setImportKeystoreState(RD.initial)
-    const sub = loadKeystore$().subscribe(setLoadKeystoreState)
+    const sub = loadKeystore$().subscribe((state) => {
+      if (RD.isSuccess(state) && !state.value) setLoadKeystoreState(RD.initial)
```

Comment:
If you have `RD.isSuccess(state)` and you have !state.value , there is something wrong. `succeed` should always have a value inside, but not `undefined`. Pls check usage / handling of `loadKeystore$` again.